Library/Formula/ipmiutil.rb

```diff
@@ -2,12 +2,12 @@
 
 class Ipmiutil < Formula
   homepage 'http://ipmiutil.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/ipmiutil/ipmiutil-2.9.1.tar.gz'
-  sha1 'f23fabe8339842fea9b8c2a601717dc002e44a9d'
+  url 'https://downloads.sourceforge.net/project/ipmiutil/ipmiutil-2.9.5.tar.gz'
+  sha1 '265f022c876da373b2ecb4be2bc0f98e65f70977'
 
-  # Make ipmiutil treat Darwin as BSD
-  patch :DATA
+  depends_on "openssl"
 
+  patch :DATA
```

Comment:
Good points. Done in e7762a1bf24d12dda45a3f385dd968114fa80916.
I submitted the upstream in http://sourceforge.net/p/ipmiutil/mailman/message/33373858/ (the link is also in the comment) — though a proper general upstream fix might be more complicated (ipmiutil's configure should detect whether MD2 is supported, not just assume it's unsupported on OS X).